package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "raml-parser",
-  "version": "0.8.18",
+  "version": "0.8.18.1",
```

Comment:
according to semver the forth block should be more descriptive as it normally is sth like alpha-3 or beta. You could add `newstore-1` to make clear that this is our patched version.